packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -247,6 +295,137 @@ enum ListTileControlAffinity {
 /// ```
 /// {@end-tool}
 ///
+/// ## The ListTile layout isn't exactly what I want
+///
+/// If the way ListTile pads and positions it's elements isn't quite what
+/// you're looking for, it's easy to create custom list items can be created
```

Comment:
The sentence after the comma seems to have too many verbs (either “it’s easy to create...” or “... can be created”)